builtin/logical/rabbitmq/backend.go

```diff
@@ -11,6 +11,8 @@ import (
 	rabbithole "github.com/michaelklishin/rabbit-hole/v2"
 )
 
+const operationPrefixRabbitMQ = "rabbit-mq"
```

Comment:
similar to github, rabbitmq is usually referred to as one word (`Rabbitmq` is OK IMO)

Reply:
+1 on using `rabbitmq`

Reply:
It looks like the `RabbitMQ` is the native capitalization for this (https://www.rabbitmq.com/). Having rabbitmq prefix will translate into something like the following in code:
```go
client.Secrets.RabbitmqReadRole(...)
```
as opposed to the following with `rabbit-mq`
```go
client.Secrets.RabbitMQReadRole(...)
```
What do you think?

Reply:
Oh I see. I didn't realize that the added `-` here affects the casing this way. `rabbit-mq` is fine then. How does it know to capitalize both letters, and not translate into `client.Secrets.RabbitMqReadRole(...)` instead?

Reply:
To capitalize both, we'd probably need some more templating magic :) Otherwise, I think you are right, it will result in a single-letter capitalization:  `client.Secrets.RabbitMqReadRole(...)`.